lib/config/config-file.js

```diff
@@ -524,7 +527,7 @@ function resolve(filePath, relativeTo) {
 function loadFromDisk(resolvedPath, configContext) {
     const dirname = path.dirname(resolvedPath.filePath),
         lookupPath = getLookupPath(dirname);
-    let config = loadConfigFile(resolvedPath);
+    let config = loadConfigFile(resolvedPath, configContext);
```

Comment:
`loadConfigFile` doesn't seem to take a second argument. Was this change deliberate?